redis.conf

```diff
@@ -1538,8 +1538,10 @@ notify-keyspace-events ""
 #
 # So use the 'requirepass' option to protect your instance.
 #
-# To enable Gopher support uncomment the following line and set
-# the option from no (the default) to yes.
+# To enable Gopher support, uncomment the following line and set the option
+# from no (the default) to yes. What's more, you must set 'io-threads-do-reads'
+# to no (the default) to avoid data race in Gopher handler, otherwise,
+# Redis will not support Gopher.
```

Comment:
I don't think we need to mention data races or anything like that here (we could have solve it if we wanted. just that it's not currently supported.
```suggestion
# Note that Ghopher is not currently supported when 'io-threads-do-reads'
# is enabled.
#
# To enable Gopher support, uncomment the following line and set the option
# from no (the default) to yes.
```